Thanks-to: Arfrever Frehtes Taifersar Arahesis <arfrever.fta@gmail.com>

--- a/cmake/FindPythonCairo.cmake
+++ b/cmake/FindPythonCairo.cmake
@@ -1,10 +1,4 @@
-if(CMAKE_VERSION VERSION_LESS 3.12.0)
-    find_package(PythonInterp 3 REQUIRED)
-    # PythonInterp sets PYTHON_EXECUTABLE
-else()
-    find_package(Python3 COMPONENTS Interpreter REQUIRED)
-    set(PYTHON_EXECUTABLE "${Python3_EXECUTABLE}")
-endif()
+find_package(PythonInterp 3 REQUIRED)
 
 # Check for python cairo
 execute_process(COMMAND ${PYTHON_EXECUTABLE} -c "import cairo"
